assets/lib/share-link/share-link.js

```diff
@@ -144,9 +132,15 @@
 		return link;
 	};
 
-	$.fn.shareLink = function( settings ) {
-		return this.each( function() {
-			$( this ).data( 'shareLink', new ShareLink( this, settings ) );
+	Element.prototype.shareLink = function ( settings ) {
+		new ShareLink( this, settings );
+		return this;
+	};
+
+	NodeList.prototype.shareLink = function ( settings ) {
```

Comment:
Couldn't we do do this above? 
I am not sure.

Reply:
@hein-obox I'm not sure I understand what you mean 🤔
As far as I understand, in order to cover all the cases that are covered by the jQuery implementation, we need to add the method to both the prototypes, what am I missing?